src/main/target/common_pre.h

```diff
@@ -251,7 +251,7 @@ extern uint8_t _dmaram_end__;
 #define USE_SERIAL_PASSTHROUGH
 #define USE_GYRO_REGISTER_DUMP  // Adds gyroregisters command to cli to dump configured register values
 #define USE_IMU_CALC
-#define USE_PPM
+//#define USE_PPM
```

Comment:
Changes in this file will not be accepted. Please remove them.

Reply:
fixed

Reply:
Still some new lines (maybe spaces)